tests/unit/components/about.spec.js

```diff
@@ -0,0 +1,62 @@
+import Vue from 'vue'
+import { mount, createLocalVue } from '@vue/test-utils'
+import Vuetify from 'vuetify'
+import VueX from 'vuex'
+import { cloneDeep } from 'lodash'
+import About from '@/views/About.vue'
+import { PEER_CONNECTED } from '@/store/mutation-types'
+import edgeDevice from '@/store/edge-device.js'
+import pnpStoreModule from '@/store/pnp'
+
+describe('About Page', () => {
+  let wrapper
+  const localVue = createLocalVue()
+  Vue.use(Vuetify)
+  localVue.use(VueX)
+
+  let store
+  const vuetify = new Vuetify()
+
+  beforeEach(() => {
+    store = new VueX.Store({
+      modules: {
+        edgeDevice: cloneDeep(edgeDevice),
+        pnp: cloneDeep(pnpStoreModule)
+      }
+    })
+
+    store.state.pnp.peerConnectionStatus = PEER_CONNECTED
+
+    wrapper = mount(About, {
+      localVue,
+      vuetify,
+      store
+    })
+  })
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  test('edgeDevice details is retrieved from store', async () => {
+    const localEdgeVersion = require('@/../package.json').version
```

Comment:
@vickywane this test is incorrect. package.json holds the PWA version, not the edge device version.